packages/pds-ember/.npmignore

```diff
@@ -0,0 +1,32 @@
+# compiled output
```

Comment:
I'm not sure if yarn honors this file, but I'm including it for good measure (generated file from `ember addon`).